harpy/_launch.py

```
@@ -212,6 +217,7 @@ def launch_snakemake(sm_args, workflow, starttext, outdir, sm_logfile, quiet, su
             sys.exit(1)
     except KeyboardInterrupt:
         # Handle a keyboard interrupt
+        testfile.close()
```

Ruff (0.8.2)

Undefined name `testfile`

(F821)